app/src/main/res/layout/fragment_review_contest.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="@dimen/activity_vertical_margin"
+    >
+
+    <TextView
```

Comment:
Once PR #28 (`gabi/con-140-overview`) is merged you can use `ContestOverviewTextViewLabel` style for this